test/multest.jl

```diff
@@ -37,3 +37,8 @@ v1 =  [0.5+0*im, 0.+0.*im]
 qv1 = normalize!(QuArray(v1))
 @assert coeffs(\(sigmax,qv1)) == [0.+0.*im, 1.+0.*im]
 @assert coeffs(\(sigmaz, sigmax)) == [0. 1.;-1. 0.]
+
+# tests for trace, dot
```

Comment:
I would prefer the tests being easier to understand on a first glance, which implies more comments. If one doesn't know about Pauli matrices it is unclear what the tests are doing. I think it is better to say for instance
```
# Pauli matrices sigmax, sigmay, sigmaz are traceless (see ... )
@assert trace(sigmax) == zero(eltype(sigmax))
@assert trace(sigmaz) == zero(eltype(sigmaz))
```
and so on.